brainstormforce/astra
# AST-1784 - Account login popup not opening on mobile/tablet

### Description
Account element Login popup form not displaying on mobile/tablet.

Also Astra Addon PR need to check - https://github.com/brainstormforce/astra-addon/pull/1026
Both are dependent

### Screenshots
https://share.getcloudapp.com/jkuvX4m1

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
Steps to reproduce

1. Add an account widget
2. On the Logged out view, please select Icon > Login Popup.
3. Visit on Desktop and click on the account, the login popup will appear
4. Now visit on mobile/tablet - this will not work.


### Checklist:
